init-data

```diff
@@ -192,6 +192,11 @@ if ! [ -f /etc/ipa/ca.crt ] ; then
 		echo "--debug" >> $OPTIONS_FILE
 	fi
 fi
+
+# check the volume-version of the bound volume, upgrade if it's different
```

Comment:
s/bound/mounted/ ?

Reply:
I guess I meant bind-mounted, I'll fix that